### PR TITLE
 NikitK/ Amending the Github page for Binary Bot  

### PR DESCRIPTION
**Pull request by NikitK-deriv**
## Changes

branding removal for other domains
https://redmine.deriv.cloud/issues/85538#NikitK/_Amending_the_Github_page_for_Binary_Bot

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/common/utils/tools.js [hunk 1]
```diff
@@ -147,3 +147,9 @@ export const errLogger = (err, msg) => {
     const errMsg = `${msg} - Error: ${errStr}`;
     console.warn(errMsg);
 };
+
+export const isBinaryDomain = !(
+    document.location.hostname !== 'localhost' &&
+    document.location.hostname !== 'bot.binary.com' &&
+    !document.location.hostname.includes('binary-bot-git-fork')
+);
```
Comment (maryia-matskevich-deriv) [inline src/common/utils/tools.js:155]: Why do we need container '!( ...)'? Why can't we specify the right variables at once ?

File: src/indexPage/react-components/logo.jsx [hunk 2]
```diff
@@ -1,16 +1,17 @@
 import React from 'react';
-import { createUrl } from '../../common/utils/tools';
+import {createUrl, isBinaryDomain} from '../../common/utils/tools';
 
-const Logo = () => (
-    <a href={createUrl({ path: '', isNonBotPage: true })} target="blank" id="logo">
+const Logo = () => {
```
Comment (maryia-matskevich-deriv) [inline src/indexPage/react-components/logo.jsx:4]: there is no need to throw a variable inside the component if the condition is to display the component or not
you can use it like this:
`const Logo = () => (<a>...</a>>)`
and then where you use it write `{isBinaryDomain && <Logo/>}`

File: templates/bot.mustache [hunk 3]
```diff
@@ -9,8 +9,14 @@
     {{> bot_css }}
 </head>
 
+<script>
+    const host_name = $(location).attr('hostname');
+    const isBinaryDomain = host_name !== 'localhos' && host_name !== 'bot.binary.com';
```
Comment (maryia-matskevich-deriv) [inline templates/bot.mustache:14]: i think you would  like to write 'localhos**t**' instead  of 'localhos'

File: templates/bot.mustache [hunk 4]
```diff
@@ -9,8 +9,14 @@
     {{> bot_css }}
 </head>
 
+<script>
```
Comment (maryia-matskevich-deriv) [inline templates/bot.mustache:12]: are you sure that we exactly need to use .mustache file? i think you can check the codebase and how we work with the same things, probably you can find better place to write it

File: templates/bot.mustache [hunk 5]
```diff
@@ -217,6 +223,14 @@
     <div class="notifyjs-corner" style="bottom: 0px; right: 0px;"></div>
     <div class="notifyjs-corner" style="bottom: 0px; left: 0px;"></div>
     </div>
+
+    <script>
+        if(location.hostname !== 'localhost' && location.hostname !== 'bot.binary.com' && !location.hostname.includes('binary-bot-git-fork')) {
```
Comment (maryia-matskevich-deriv) [inline templates/bot.mustache:228]: here the same

File: templates/index.mustache [hunk 6]
```diff
@@ -66,5 +66,13 @@
     {{> index}}
     {{> gtm_iframe }}
    </div>
+    <script>
+        if(location.hostname !== 'localhost' && location.hostname !== 'bot.binary.com' && !location.hostname.includes('binary-bot-git-fork')) {
```
Comment (maryia-matskevich-deriv) [inline templates/index.mustache:70]: here the same

**Review by maryia-matskevich-deriv — CHANGES_REQUESTED** [review]
please check suggestions and questions

File: src/common/utils/tools.js (hunk 1 above)
Comment (NikitK-deriv) [reply src/common/utils/tools.js]: I used this approach which once was approved by Yashim as part of branding removal for other projects. In this case we can change it. Please check again 🙏

File: src/indexPage/react-components/logo.jsx (hunk 2 above)
Comment (NikitK-deriv) [reply src/indexPage/react-components/logo.jsx]: When I met this place first time I was a little bit confused can we use logic here or not. But it works. Thanks 👍 

Screenshot 2023-01-24 at 14 22 49

File: templates/bot.mustache (hunk 3 above)
Comment (NikitK-deriv) [reply templates/bot.mustache]: It's just a chore. Removed, thanks

File: templates/bot.mustache (hunk 4 above)
Comment (NikitK-deriv) [reply templates/bot.mustache]: So `src` for images hardcoded in mustache template
If we have any way to resolve it or some examples, please share where I can find it @maryia-matskevich-deriv 🙏 
Unfortunately I didn't found any other cases

I only noticed that we use build strategy to build path for src, but we need a `window.location` information in web page to resolve this logic.

File: src/common/utils/tools.js [hunk 7]
```diff
@@ -147,3 +147,8 @@ export const errLogger = (err, msg) => {
     const errMsg = `${msg} - Error: ${errStr}`;
     console.warn(errMsg);
 };
+
+export const isBinaryDomain =
```
Comment (maryia-matskevich-deriv) [inline src/common/utils/tools.js:151]:
```suggestion
export const is_binary_domain =
```

**Review by maryia-matskevich-deriv — CHANGES_REQUESTED** [review]
check a suggestion

**Review by sandeep-deriv — APPROVED** [review]

**Review by prince-deriv — APPROVED** [review]

**Pull request merged**